WalletWasabi.Packager/Program.cs

```diff
@@ -749,14 +749,24 @@ private static void Publish()
 					Directory.Move(publishedFolder, newFolderPath);
 					publishedFolder = newFolderPath;
 
+					var commands = new[] {
+						"cd ~",
+						"sudo umount /mnt/c",
+						"sudo mount -t drvfs C: /mnt/c -o metadata",
+						$"cd /mnt/c/Users/user/Desktop/WalletWasabi/WalletWasabi.Gui/bin/dist",
+						$"sudo find ./{newFolderName} -type f -not -name 'wassabee' -exec chmod 644 {{}} \\;",
+						$"tar -pczvf {newFolderName}.tar.gz {newFolderName}"
+					};
+					string arguments = string.Join(" && ", commands);
```

Comment:
@lontivero This ruins the deterministic build guide if this is not on my computer.

Reply:
@nopara73 I didn't change any path, it is hardcoded  since ever.

Reply:
The path wasn't hardcoded here.